src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -2149,6 +2149,7 @@ def load_arguments(self, _):
         c.argument('ssh_key', help='SSH key file location for SSH connections.', options_list=['--ssh-key'])
     with self.argument_context('network bastion rdp') as c:
         c.argument('disable_gateway', arg_type=get_three_state_flag(), help='Flag to disable access through RD gateway.')
+        c.argument('enable_mfa', help='Enable RDS auth for MFA if supported by the target machine.')
```

Comment:
It's better to leverage https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags.

Reply:
done